README.md

```diff
@@ -132,3 +132,5 @@ class FelineAdmin(TypedModelAdmin):
 
 * Django 3.1+
 * Python 3.6+
+
+Django 4.1 is now supported (unique constraints defined on Meta are not tested)
```

Comment:
I think this might be counterproductive in the readme; I'd just remove this line. the above is already accurate (django 3.1+ is supported). For version support for historic versions, users can look at the Releases page to see when support for various versions was added.

> unique constraints defined on Meta are not tested

I don't see any reason why they wouldn't work though; I wouldn't warn about this. If they don't work, someone can file an issue :)